diff --git a/src/index.ts b/src/index.ts
index 35d2757a46df570e20dd0002f2bcff22dfa3fb5f..3e58a9142a7f230c583cacafbac92407a44a7f2a 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -56,10 +56,6 @@ export class MSafeWalletAdapter implements AdapterPlugin {
         .catch((e) => {
           console.log("🚀 ~ file: index.ts:57 ~ MSafeWalletAdapter:", e);
         });
-    } else {
-      console.log(
-        "🚀 ~ file: index.ts:63 ~ MSafeWalletAdapter: not in msafe wallet:"
-      );
     }
   }
 
