diff --git a/src/index.ts b/src/index.ts
index ea19ab90c0a3add8447d6006d611ffb120f9662c..3da90e1647208afaef1b46f09f5e4f87349aebc3 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -60,9 +60,6 @@ export class MSafeWalletAdapter implements AdapterPlugin {
           console.log("🚀 ~ file: index.ts:57 ~ MSafeWalletAdapter:", e);
         });
     } else {
-      console.log(
-        "🚀 ~ file: index.ts:63 ~ MSafeWalletAdapter: not in msafe wallet:"
-      );
     }
   }
   url = "https://aptos.m-safe.io/";
