diff --git a/dist/index.js b/dist/index.js
index 12fe9afc73a78db35154177abfcc3d68270f778e..e79709ae0c158f2f08644bbfeb43073cca228554 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1015,7 +1015,6 @@ function useIsMounted() {
 import { useBalance } from "wagmi";
 
 // src/hooks/useMainnetEnsAvatar.ts
-import { normalize } from "viem/ens";
 import { useEnsAvatar } from "wagmi";
 import { mainnet as mainnet2 } from "wagmi/chains";
 
@@ -1033,15 +1032,9 @@ function useIsMainnetConfigured() {
 // src/hooks/useMainnetEnsAvatar.ts
 function useMainnetEnsAvatar(name) {
   const mainnetConfigured = useIsMainnetConfigured();
-  const safeNormalize = (ensName) => {
-    try {
-      return normalize(ensName);
-    } catch {
-    }
-  };
   const { data: ensAvatar } = useEnsAvatar({
     chainId: mainnet2.id,
-    name: name ? safeNormalize(name) : void 0,
+    name,
     query: {
       enabled: mainnetConfigured
     }
