diff --git a/node_modules/hardhat/internal/core/jsonrpc/types/output/transaction.js b/node_modules/hardhat/internal/core/jsonrpc/types/output/transaction.js
index 2d383ce..dbc1d45 100644
--- a/node_modules/hardhat/internal/core/jsonrpc/types/output/transaction.js
+++ b/node_modules/hardhat/internal/core/jsonrpc/types/output/transaction.js
@@ -37,9 +37,9 @@ exports.rpcTransaction = t.type({
     to: io_ts_1.optional(io_ts_1.nullable(base_types_1.rpcAddress)),
     transactionIndex: io_ts_1.nullable(base_types_1.rpcQuantity),
     value: base_types_1.rpcQuantity,
-    v: base_types_1.rpcQuantity,
-    r: base_types_1.rpcQuantity,
-    s: base_types_1.rpcQuantity,
+    v: io_ts_1.nullable(base_types_1.rpcQuantity),
+    r: io_ts_1.nullable(base_types_1.rpcQuantity),
+    s: io_ts_1.nullable(base_types_1.rpcQuantity),
     // EIP-2929/2930 properties
     type: io_ts_1.optional(base_types_1.rpcQuantity),
     chainId: io_ts_1.optional(io_ts_1.nullable(base_types_1.rpcQuantity)),
diff --git a/node_modules/hardhat/src/internal/core/jsonrpc/types/output/transaction.ts b/node_modules/hardhat/src/internal/core/jsonrpc/types/output/transaction.ts
index 7b315ce..2726703 100644
--- a/node_modules/hardhat/src/internal/core/jsonrpc/types/output/transaction.ts
+++ b/node_modules/hardhat/src/internal/core/jsonrpc/types/output/transaction.ts
@@ -19,9 +19,9 @@ export const rpcTransaction = t.type(
     to: optional(nullable(rpcAddress)),
     transactionIndex: nullable(rpcQuantity),
     value: rpcQuantity,
-    v: rpcQuantity,
-    r: rpcQuantity,
-    s: rpcQuantity,
+    v: nullable(rpcQuantity),
+    r: nullable(rpcQuantity),
+    s: nullable(rpcQuantity),
 
     // EIP-2929/2930 properties
     type: optional(rpcQuantity),
